src/app/shared/components/keyboard-input/keyboard-input.component.scss

```diff
@@ -17,6 +17,10 @@ input {
   font-weight: lighter;
 }
 
+sofie-row {
+  justify-content: space-between;
+}
```

Comment:
This goes a bit against the reusable component approach, since `justify-content` couples to the implementation detail that it is a `flex` (or `grid`) container. If we change how `sofie-row` produces the row to e.g. a new CSS API or use `float` on all immediate children, then all the places that do overwrites like this, are likely to be broken.
So either this functionality should be embedded into the RowComponent as an input or we should evaluate if we need the row component.

Reply:
I added it as Boy Scout rule, but maybe we need separate task and more logic for that. For that reason I remove it for now.